lib/webserver/ApiRouter.js

```diff
@@ -27,6 +27,7 @@ const removeOldUploads = async (extension = null) => {
  * Api routes used by the webinterface
  *
  * @param {import("./WebServer")} webserver
+ * @returns {import("express").Router}
```

Comment:
Type is already imported so `@returns {express.Router}` should just work, right?

Reply:
that's right